scripts/tgtg_auth.py

```
@@ -0,0 +1,87 @@
+        _result["creds"] = creds
+        _result["client"] = client
+        _write_state("SUCCESS", {"msg": "Tokens received!"})
+    except Exception as exc:
+        _error.append(str(exc))
+        _write_state("ERROR", {"msg": str(exc)})
+    finally:
+        _done.set()
+
+
+print(f"Starting TGTG auth flow for {EMAIL}", flush=True)
+print(f"This will send a magic link email to that address.", flush=True)
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)